Makefile

```diff
@@ -146,11 +146,11 @@ else
 endif
 
 build-docs:
-	cargo doc --release $(compiler_features) --document-private-items --no-deps --workspace
+	cargo doc --release --no-deps --manifest-path lib/api/Cargo.toml
 
 build-docs-capi:
 	cd lib/c-api/doc/deprecated/ && doxygen doxyfile
-	cargo doc --manifest-path lib/c-api/Cargo.toml --no-deps --features wat,jit,object-file,native,cranelift,wasi
+	cargo doc --release --no-deps --manifest-path lib/c-api/Cargo.toml
```

Comment:
I think this probably isn't what we want, disabling those features means those features won't show up in the generated docs. I think we should at least have the same features enabled as what we release with